.ci/docker/common/install_amdsmi.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -ex
+
+cd /opt/rocm/share/amd_smi && pip install .
```

Comment:
Does this require specific rocm version? I didn't see amd_smi in the internal rocm version we have (6.0.0):

```
$ ls /opt/rocm/share/
amd_smi  doc  hip  hipfort  html  info  man  miopen  mivisionx  openmp-extras  pkgconfig  rccl  rocgdb  rocm  rocm_smi  rocprofiler
```

Reply:
AMD SMI is packaged with ROCm since 5.6, and should come in with `amdgpu-install --usecase=rocm` or as we installed here in install_rocm.sh with amd-smi-lib